nuxt.config.ts

```diff
@@ -11,6 +11,14 @@ export default defineNuxtConfig({
     host: '0.0.0.0',
   },
 
+  postcss: {
+    plugins: {
+      'tailwindcss/nesting': {},
+      tailwindcss: { config: './libs/ui/tailwind.config.js' },
```

Comment:
what if we move this package outside repo? will it still work?

Reply:
Right now we can't do that because UI kit doesn't have its own build. If we want to do this, we need to add a separate build stage for UI kit, if we want to do this, I think it is better to do it as a separate task